internal/app/machined/pkg/controllers/config/k8s_control_plane.go

```diff
@@ -16,11 +16,11 @@ import (
 	"github.com/talos-systems/os-runtime/pkg/resource"
 	"github.com/talos-systems/os-runtime/pkg/state"
 
-	"github.com/talos-systems/talos/internal/app/machined/pkg/resources/config"
 	"github.com/talos-systems/talos/pkg/images"
 	talosconfig "github.com/talos-systems/talos/pkg/machinery/config"
 	"github.com/talos-systems/talos/pkg/machinery/config/types/v1alpha1/machine"
 	"github.com/talos-systems/talos/pkg/machinery/constants"
+	"github.com/talos-systems/talos/pkg/resources/config"
```

Comment:
moved resources packages to public, as they are part of the public interface to some extent (they contain resource definitions, types, etc.) 

`pkg/cluster/kubernetes` used for conversion depends on resources package, so to keep imporvet rules we need to make this package public